### ISSUE TITLE
Generate code for duplicated NestedSDFG only once

### ISSUE THREAD

**Issue by TizianoDeMatteis** [spcl/dace#392]
It would be nice to have a way to avoid generating multiple times code for a nested SDFG that appears multiple times in the SDFG.

- It must be understood how to detect duplicated NestedDFG: for example, we can add another property to the SDFG, such as "unique_name" or something like this

- how to trigger this behavior: we may don't want to do this always
